app/services/api/v3/participant_declarations_query.rb

```diff
@@ -75,25 +75,8 @@ def npq_lead_provider
       end
 
       def declarations_scope
-        scope = ParticipantDeclaration.for_lead_provider(cpd_lead_provider)
-
-        if cohort_years.present?
-          scope = ecf_cohort_for(scope).or(npq_cohort_for(scope))
-        end
-
-        scope
-      end
-
-      def ecf_cohort_for(scope)
-        return ParticipantDeclaration.none if lead_provider.blank?
-
-        with_joins(scope).where(participant_profile: { induction_records: { cohorts: { start_year: cohort_years } } })
-      end
-
-      def npq_cohort_for(scope)
-        return ParticipantDeclaration.none if npq_lead_provider.blank?
-
-        with_joins(scope).where(participant_profile: { type: "ParticipantProfile::NPQ", schedule: { cohorts: { start_year: cohort_years } } })
+        scope = with_joins(ParticipantDeclaration.for_lead_provider(cpd_lead_provider))
```

Comment:
do we need the `with_joins` if there is no cohort filter? also worth revisiting those joins as we might no longer need some of them